contracts/staking/contracts/src/vaults/StakingPoolRewardVault.sol

```diff
@@ -50,6 +52,9 @@ contract StakingPoolRewardVault is
     // mapping from Pool to Reward Balance in ETH
     mapping (bytes32 => Balance) internal balanceByPoolId;
 
+    // address of ether vault
+    IEthVault internal ethVault;
```

Comment:
Ah my bad, I didn't realize we were prefixing with `_` for internal state variables. This will have to change throughout the staking contracts. I'm going to leave it consistent for now but created a task to update all of our internal state variables.